src/build/patches.ts

```diff
@@ -142,6 +170,31 @@ function handleProperty(child: Node): Partial<Property> {
   };
 }
 
+/**
+ * Handles a child node of type "method" and adds it to the method object.
+ * @param child The child node to handle.
+ */
+function handleMethod(child: Node): Partial<Method> {
+  const name = child.values[0] as string;
+  const type = child.children[0];
```

Comment:
We should iterate and check types than assuming a position or using `filter()`, otherwise we can't throw on unexpected child. Just like line 120.